#
# Definition of all trackable objects
# Identifier corresponds to Trackable ID set in Tracking Tools
#
mocap_node:
    ros__parameters:
        rigid_bodies:
            6:
                pose: "ph1/pose"
                pose2d: "ph1/ground_pose"
                odom: ph1/Odom
                tf: tf
                child_frame_id: "ph1/base_link"
                parent_frame_id: "world"
            7:
                pose: "ph2/pose"
                pose2d: "ph2/ground_pose"
                odom: ph2/Odom
                tf: tf
                child_frame_id: "ph2/base_link"
                parent_frame_id: "world"
        optitrack_config:
                multicast_address: "239.255.42.99"
                command_port: 1510
                data_port: 1511
                enable_optitrack: true
                version: [4, 0, 0, 0]
